data/fleets.txt

```diff
@@ -1233,6 +1233,142 @@ fleet "Navy Surveillance"
 		"Surveillance Drone" 4
 	variant 10
 		"Gunboat" 2
+	
+fleet "Navy Logistics"
+	government "Republic"
+	names "republic capital"
+	fighters "republic fighter"
+	cargo 0
+	personality
+		heroic opportunistic
+	variant 5
+		"Auxiliary (Cargo)"
+		"Combat Drone" 2
+		"Lance" 
+		"Dropship" 
+		"Frigate"
+		"Rainmaker"
+		"Gunboat" 2
+	variant 4
+		"Auxiliary"
+		"Combat Drone" 2
+		"Dropship" 2
+		"Frigate"
+		"Rainmaker"
+		"Gunboat"
+	variant 3
+		"Auxiliary (Transport)"
+		"Dropship" 2
+		"Surveillance Drone"
+		"Combat Drone"
+		"Frigate"
+		"Rainmaker" 2
+		"Gunboat"2
+	variant 2
+		"Auxiliary"
+		"Combat Drone" 2
+		"Lance" 2
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"
+	variant 2
+		"Auxiliary"
+		"Combat Drone" 2
+		"Dropship" 2
+		"Cruiser"
+		"Combat Drone" 4
+	variant 1
+		"Auxiliary (Cargo)" 2
+		"Combat Drone" 4
+		"Dropship" 2
+		"Lance" 2
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"
+	variant 1
+		"Auxiliary (Transport)" 2
+		"Dropship" 4
+		"Combat Drone" 4
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"2
+	variant 1
+		"Auxiliary (Transport)"
+		"Dropship" 2
+		"Combat Drone" 2
+		"Auxiliary (Cargo)"
+		"Combat Drone" 2
+		"Dropship" 
+		"Lance" 
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"2
+	variant 1
+		"Auxiliary"
+		"Dropship" 2
+		"Combat Drone" 2
+		"Auxiliary (Cargo)"
+		"Combat Drone" 2
+		"Dropship" 
+		"Lance" 
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"
+	variant 1
+		"Auxiliary"
+		"Dropship" 2
+		"Combat Drone" 2
+		"Auxiliary (Transport)"
+		"Dropship" 2
+		"Combat Drone" 2
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"2
+	variant 1
+		"Auxiliary (Cargo)" 2
+		"Combat Drone" 4
+		"Dropship" 2
+		"Lance" 2
+		"Cruiser" 2
+		"Combat Drone" 8
+	variant 1
+		"Auxiliary" 2
+		"Dropship" 2
+		"Lance" 2
+		"Combat Drone" 4
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate" 2
+		"Rainmaker"
+		"Gunboat"2
+	variant 1
+		"Auxiliary (Transport)"
+		"Dropship" 2
+		"Combat Drone"
+		"Surveillance Drone"
+		"Auxiliary"
+		"Dropship" 2
+		"Combat Drone" 2
+		"Auxiliary (Cargo)"
+		"Combat Drone" 2
+		"Dropship" 
+		"Lance" 
+		"Carrier"
+		"Lance" 4
+		"Combat Drone" 6
+		"Cruiser"
+		"Combat Drone" 4
+		"Frigate"
```

Comment:
A carrier, a cruiser, a frigate, and three (heavy) auxiliaries; are you sure this is merely logistics and not a full armada?